docs/docsite/rst/network_aggregate_resources.rst

```diff
@@ -0,0 +1,175 @@
+*********************************
+Aggregate Resources in Networking
+*********************************
+
+.. contents:: Topics
+
+This section explores how and when `Aggregate Resources` can be used within Ansible Networking.
```

Comment:
Please don't capitalize "Aggregate Resources" unless it's a proper name or a trademark. Re: 'Ansible Networking' - in most cases (unless it's a trademark or proper name) 'networking' shouldn't be capitalized.